tests/test_admin_cli.py

```diff
@@ -239,7 +239,7 @@ def test_do_status(self):
         output = temp_output.getvalue()
 
         states_text = "| States       |              738 |"
-        contacts_text = "| Contacts     |               15 |"
```

Comment:
Hmm. Why has this change increased the number of contacts recorded in the database?

Reply:
Because we started to skip empty rows in Replay Importers and `sen_frig_sensor.dsf` had only errors due to that. It is imported by Replay Contact Importer now, and the number of Contacts increases. (I'm adding a note for that)